extmod/uasyncio/core.py

```diff
@@ -215,20 +234,41 @@ def run(coro):
 
 
 class Loop:
-    def create_task(self, coro):
+    @staticmethod
+    def create_task(coro):
         return create_task(coro)
 
-    def run_forever(self):
+    @staticmethod
+    def run_forever():
         run_until_complete()
         # TODO should keep running until .stop() is called, even if there're no tasks left
 
-    def run_until_complete(self, aw):
+    @staticmethod
+    def run_until_complete(aw):
         return run_until_complete(_promote_to_task(aw))
 
-    def close(self):
+    @staticmethod
+    def close():
         pass
 
+    @staticmethod
+    def set_exception_handler(handler):
+        global _exc_handler
+        _exc_handler = handler
+
+    @staticmethod
+    def get_exception_handler():
+        return _exc_handler
+
+    @classmethod
+    def default_exception_handler(cls, context):
+        _default_exc_handler(cls, context)
```

Comment:
How about putting the implementation of `_default_exc_handler` right here?  That would avoid defining this function above and save RAM (each function definition uses RAM).